client/src/components/PublicPages/AuditPlanner/AuditPlanner.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { H1 } from '@blueprintjs/core'
+import { toast } from 'react-toastify'
+
+import ElectionResultsCard from './ElectionResultsCard'
+import {
+  constructInitialElectionResults,
+  IElectionResults,
+  validateElectionResults,
+} from './electionResults'
+import { Inner } from '../../Atoms/Wrapper'
+
+const PageHeading = styled(H1)`
+  margin-bottom: 24px;
+`
+
+const AuditPlanner: React.FC = () => {
+  const [electionResults, setElectionResults] = useState<IElectionResults>(
+    constructInitialElectionResults()
+  )
+  const [electionResultsEditable, setElectionResultsEditable] = useState(true)
+
+  const planAudit = () => {
+    try {
+      validateElectionResults(electionResults)
+    } catch (err) {
+      toast.error(err instanceof Error ? err.message : 'Unknown error')
+      return
+    }
+    setElectionResultsEditable(false)
```

Comment:
In my next PR, I'll continue building out the `planAudit` handler to actually compute sample sizes